packages/zoe/src/issuerStorage.js

```diff
@@ -1,22 +1,32 @@
 // @ts-check
 
-import { makeWeakStore } from '@agoric/store';
+import { provideDurableWeakMapStore } from '@agoric/vat-data';
 import { E } from '@endo/eventual-send';
 
 import { arrayToObj } from './objArrayConversion.js';
 import { cleanKeywords } from './cleanProposal.js';
 import { makeIssuerRecord } from './issuerRecord.js';
 
+const STORAGE_INSTANTIATED_KEY = 'IssuerStorageInstantiated';
+
 /**
- *  Make the Issuer Storage.
+ * Make the Issuer Storage.
+ *
+ * @param {import('@agoric/vat-data').Baggage} zcfBaggage
  */
-export const makeIssuerStorage = () => {
+export const makeIssuerStorage = zcfBaggage => {
```

Comment:
```suggestion
export const provideIssuerStorage = zcfBaggage => {
```
not `vivify` because no kind behaviors are reestablished. But not `make` because it is more about providing rather than making new ones.

Reply:
done